src/superphot_plus/classify_ztf.py

```diff
@@ -113,62 +119,33 @@ def run_single_fold(x):
         val_classes = SnClass.get_classes_from_labels(val_labels)
         test_classes = SnClass.get_classes_from_labels(test_labels)
 
-        train_chis = calculate_neg_chi_squareds(train_names, FITS_DIR, DATA_DIRS)
-        train_features, train_classes, train_chis = oversample_using_posteriors(
-            train_names, train_classes, train_chis, goal_per_class
-        )
-        val_chis = calculate_neg_chi_squareds(val_names, FITS_DIR, DATA_DIRS)
-        val_features, val_classes, val_chis = oversample_using_posteriors(
-            val_names, val_classes, val_chis, round(0.1 * goal_per_class)
-        )
-
-        train_features = np.append(
-            train_features,
-            np.array(
-                [
-                    train_chis,
-                ]
-            ).T,
-            1,
+        #train_chis = calculate_neg_chi_squareds(train_names, FITS_DIR, DATA_DIRS)
+        train_features, train_classes = oversample_using_posteriors(
+            train_names, train_classes, goal_per_class
         )
-        val_features = np.append(
-            val_features,
-            np.array(
-                [
-                    val_chis,
-                ]
-            ).T,
-            1,
+        #val_chis = calculate_neg_chi_squareds(val_names, FITS_DIR, DATA_DIRS)
+        val_features, val_classes = oversample_using_posteriors(
+            val_names, val_classes, round(0.1 * goal_per_class)
         )
 
         test_features = []
         test_classes_os = []
         test_group_idxs = []
         test_names_os = []
-        test_chis_os = []
-        test_chis = calculate_neg_chi_squareds(test_names, FITS_DIR, DATA_DIRS)
 
         for i in range(len(test_names)):
             test_name = test_names[i]
-            test_posts = get_posterior_samples(test_name)
+            test_posts = get_posterior_samples(test_name, FITS_DIR, "dynesty")
```

Comment:
Quick question! Will we always use the dynasty sampler to get the posteriors, or should we make it so that the sampler might be flexible (dynesty / nuts / svi)?

Reply:
The other samplers haven't been updated to include the chisq values in their returned posteriors yet, so for now I'm restricting them to all dynesty. But this restriction will be removed later!